### ISSUE TITLE
The issue of negative loss values.

### ISSUE THREAD

**Issue by Cresynia** [CEA-LIST/MonoProb#4]
Thanks！And I met another problem when I train it. Is it normal for the loss to be a negative number？
![image](https://github.com/CEA-LIST/MonoProb/assets/90834089/34299023-3ee3-4af1-8bcf-7bb5e49ac37a)

And the warning said:
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
And I found that the loss is negative,  could it be related to the learning rate, and should I consider swapping their positions?or I need to pay attention to some other aspects? Thanks for your assistance.

**Comment by marsalr** [comment]
Yes this is normal. The effect of swapping `lr_scheduler.step()` and `optimizer.step()` on the training should be negligible.

**Comment by illuosion** [comment]
I have the same problem as the loss is negative
